history/5279.md

```diff
@@ -0,0 +1 @@
+* @barnson: Enable the WiX native SDK when the new Visual C++ Build Tools bundle is installed. Fixes wixtoolset/issues#5279.
```

Comment:
Commit comment works. See aae66628e918a5e2c98232134225fb01266370de. (And it's a good thing because it means we can do it offline.)